### PR TITLE
started adding gui

### PR DESCRIPTION
**Pull request by clrosenbluth**

### CODE REVIEW

File: src/main/java/ScrabbleGame.java [hunk 1]
```diff
@@ -25,16 +25,17 @@ public ScrabbleGame(
...
-        {
-            removeTiles(word);
-            addWord(word);
-            addTiles();
-            return true;
-        }
-        return false;
+        word = word.toUpperCase();
+
+        if (!isWord(word)) throw new NotAWordException();
```
Comment (andrewoid) [inline src/main/java/ScrabbleGame.java]: Always use {}s

File: src/test/java/ScrabbleGameTest.java [hunk 2]
```diff
@@ -22,7 +22,14 @@ public void playWord_true()
...
         // when
-        boolean val = game.playWord(word);
+        boolean val;
+        try
+        {
+            val = game.playWord(word);
+        } catch (Exception e)
+        {
+            val = false;
+        }
```
Comment (andrewoid) [inline src/test/java/ScrabbleGameTest.java]: You don't need the try/catch.
Just declare that the test method throws the Exceptions.

File: src/main/java/ScrabbleGame.java [hunk 3]
```diff
@@ -25,16 +25,17 @@ public ScrabbleGame(
...
+        if (!hasTiles(word)) throw new InsufficientTilesException();
+
+        removeTiles(word);
+        addWord(word);
+        addTiles();
+        return true;
```
Comment (andrewoid) [inline src/main/java/ScrabbleGame.java:49]: This method doesn't need to return a boolean anymore.

File: src/test/java/ScrabbleGameTest.java (hunk 2 above)
Comment (clrosenbluth) [reply src/test/java/ScrabbleGameTest.java]: When I do this, my tests say that they fail. How can I catch an exception being thrown by a test?

Comment (andrewoid) [reply src/test/java/ScrabbleGameTest.java]: This test method shouldn't be throwing an exception, right?

Comment (clrosenbluth) [reply src/test/java/ScrabbleGameTest.java]: So then what do you mean by "declare that the test method throws the Exceptions"?

Comment (andrewoid) [reply src/test/java/ScrabbleGameTest.java]: public void methodName() throws SpecificException

**Review by andrewoid — APPROVED** [review]

**Pull request merged**